.github/workflows/prerelease.yml

```diff
@@ -52,35 +66,9 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ steps.setup-env.outputs.python-version }}
-      - uses: actions/cache@v3
-        env:
-          BASE_CACHE_KEY: "${{ github.job }}-${{ runner.os }}-\
-            py${{ steps.setup-python.outputs.python-version }}-\
-            packer${{ steps.setup-env.outputs.packer-version }}-\
-            tf-${{ steps.setup-env.outputs.terraform-version }}-"
+      - uses: hashicorp/setup-packer@v3
```

Comment:
Same thing here.

Reply:
Please see commit [6232a19](https://github.com/cisagov/skeleton-packer/pull/376/commits/6232a1944f9793b244bc0ab7e20c9aca854553a8).

Reply:
Is there a reason why this conversation isn't resolved?  I think I took care of it just as was done in `release.yml`.

Reply:
Nope, I must've just missed this conversation for some reason when I went through. Sorry about that!